server/tests/unit/common/apis/test_api_v3_sparse.py

```diff
@@ -0,0 +1,916 @@
+from urllib.parse import quote
```

Comment:
I'm not sure if fully duplicating `test_api_v3(_dense).py` is the way to go. Let me give this some thought.

Reply:
Actually, I think you're right...It's inconvenient to have to add/modify a test in two places